mono/mini/mini-posix.c

```diff
@@ -825,10 +825,14 @@ native_stack_with_gdb (pid_t crashed_pid, const char **argv, FILE *commands, cha
 	argv [3] = commands_filename;
 	argv [4] = "-nx";
 
-	fprintf (commands, "attach %ld\n", (long) crashed_pid);
-	fprintf (commands, "info threads\n");
-	fprintf (commands, "thread apply all bt\n");
-
+	fprintf (commands, "attach %ld\n"
```

Comment:
Fewer calls to fprintf, smaller code.

Reply:
This is changing code for changing code. Changing the number of calls to fprintf doesn't fix anything, it just looks better.

Reply:
Either way is fine.

Reply:
It looks smaller, fewer function calls, less code, same functionality, so better.

Reply:
The improvement is really marginal here, so this is really just a style change.